apps/pie-microsite/src/_11ty/shortcodes/contentPageImage.js

```diff
@@ -17,20 +24,22 @@ const createCaption = config => (config.caption
 module.exports = function (config) {
     const context = config.context ?? 'contentPage';
     const contextClass = `c-${context}-img`;
+    const isImageFullContainerWidth = !config.width;
+    const imageStyles = !isImageFullContainerWidth ? `style="--img-width: ${config.width};"` : ''; // If image isn't full width, set it to required width
+    const imageAlt = config.alt ? `alt="${config.alt}"` : '';
+    const figureClasses = [
+        contextClass,
+        'c-contentImage',
+        ...(!isImageFullContainerWidth ? ['c-contentImage--hasBackdrop'] : [])
+    ];
 
-    if (config.type !== 'spread') {
-        return `<figure class="${contextClass} c-contentImage c-contentImage--with-bg">
-            <div class="c-contentImage-bg">
-              <img style="--img-width: ${config.width};" src="${config.src}" ${
-          config.alt ? `alt="${config.alt}"` : ''
-        } />
-            </div>${createCaption(config)}
-          </figure>`;
-    }
-
-    return `<figure class="${contextClass} c-contentImage">
-        <img src="${config.src}" ${
-      config.alt ? `alt="${config.alt}"` : ''
-    } />${createCaption(config)}
+    return `<figure class="${figureClasses.join(' ')}">
+        <div class="c-contentImage-backdrop">
+          <picture>
+            ${config.mobileSrc ? `<source ${imageStyles} media="(max-width: 600px)" srcset="${config.mobileSrc}">` : ''}
```

Comment:
perhaps we should define the 600px as a cont above with a comment explaining why (its the narrowMid value in fozzie)